tests/libcxx/expected_results.txt

```diff
@@ -509,25 +509,17 @@ std/utilities/smartptr/unique.ptr/unique.ptr.class/unique.ptr.asgn/move_convert.
 std/utilities/smartptr/unique.ptr/unique.ptr.class/unique.ptr.asgn/move_convert.single.pass.cpp FAIL
 std/utilities/smartptr/unique.ptr/unique.ptr.class/unique.ptr.asgn/move.pass.cpp FAIL
 
-# Not analyzed, likely bogus tests. Appears to be timing assumptions.
+# `ALLOW_RETRIES` comments indicate tests with timing assumptions

```

Comment:
The change markup isn't great. There are actually no removals here, I simply pulled the skips with `ALLOW_RETRIES` comments out into a new section, and added two more on green lines 515 and 522. The other red lines here are exactly the other green lines below.

Reply:
Yep, double-checked the partition - thanks! :heart_eyes_cat: